torchtext/vocab.py

```diff
@@ -442,6 +448,17 @@ def __init__(self, name='840B', dim=300, **kwargs):
         super(GloVe, self).__init__(name, url=url, **kwargs)
 
 
+class Word2Vec(Vectors):
```

Comment:
Please add some docs here (including examples) and unit tests. Thanks for the contribution.

Reply:
Thank you for your comment. I will add examples to [`vocab.rst` file later.